### PR TITLE
Add context path for jaeger, added grafana urls

### PR DESCRIPTION
**Pull request by brendarearden**
**Issue**
Kiali was showing error in dashboard indicating that it was unable to get Jaeger traces

**Solution**
Update Jaeger URL and Grafana URL to the correct urls to ensure Kiali has access to them

**Issue**
https://github.com/rancher/rancher/issues/30752

### CODE REVIEW

**Comment by nickgerace** [comment]
Most likely will need an RC version bump. @aiyengar2 is working on a PR to add this to the docs!

**Comment by brendarearden** [comment]
I PMed this to @arvind, but it might be a good question for you to get the answer to since you are documenting it @nickgerace: The istio chart does not have a package.yaml file since it does not have an upstream chart to pull from (so we just never added it). Do I need to add one with just the release candidate value in it, so that it works for validation?

**Comment by nickgerace** [comment]
@brendarearden: as far as I understand, _yes_. Updating the RC version should be enough to satisfy validation, regardless of how the `package.yaml` is set up. Though, @aiyengar2 would know more here :)

**Comment by brendarearden** [comment]
🤦 I forgot to fetch upstream - there was a package.yaml already on the chart from arvind's changes and I see my mistakes. we should be good to go now

**Comment by arvind** [comment]
Hi all, although I do indeed work on charts and data visualizations, I don't believe I'm the right Arvind for this issue :)

**Review by nickgerace — APPROVED** [review]

**Review by aiyengar2 — APPROVED** [review]
LGTM

File: packages/rancher-istio/charts/values.yaml [hunk 1]
```diff
@@ -90,10 +90,14 @@ kiali:
       custom_metrics_url: "http://rancher-monitoring-prometheus.cattle-monitoring-system.svc:9090"
       url: "http://rancher-monitoring-prometheus.cattle-monitoring-system.svc:9090"
     tracing:
-      in_cluster_url: "http://tracing.istio-system.svc:16686"
+      in_cluster_url: "http://tracing.istio-system.svc:16686/jaeger"
+    grafana:
+      in_cluster_url: "http://rancher-monitoring-grafana.cattle-monitoring-system.svc:80"
+      url: "http://rancher-monitoring-grafana.cattle-monitoring-system.svc:80"
```
Comment (cbron) [inline packages/rancher-istio/charts/values.yaml:96]: For these things, need to add them to app-readme and any other docs location.

**Review by nickgerace — APPROVED** [review]

File: packages/rancher-istio/charts/README.md [hunk 2]
```diff
@@ -14,26 +14,42 @@ A Rancher created chart that packages the istioctl binary to install via a helm
...
+```
+The url depends on the default values for `nameOverride`, `namespaceOverride`, and `prometheus.service.port` being set in your rancher-monitoring or other monitoring instance.
+
+##### Grafana
+The `kiali.external_services.grafana` url is set in the values.yaml:
+```
+http://{{ .Values.nameOverride }}-grafana.{{ .Values.namespaceOverride }}.svc:{{ grafana.service.port }}
+```
+The url depends on the default values for `nameOverride`, `namespaceOverride`, and `grafana.service.port` being set in your rancher-monitoring or other monitoring instance.
+##### Tracing
```
Comment (cbron) [inline packages/rancher-istio/charts/README.md:46]: Nit: newline above this, but doesn't affect display of markdown.

File: packages/rancher-istio/charts/README.md [hunk 3]
```diff
@@ -14,26 +14,42 @@ A Rancher created chart that packages the istioctl binary to install via a helm
...
-The url depends on the default values for `nameOverride`, `namespaceOverride`, and `prometheus.service.port` being set in your rancher-monitoring or other monitoring instance.
-
 The Monitoring app sets `prometheus.prometheusSpec.ignoreNamespaceSelectors=false` which means all namespaces will be scraped by Prometheus by default. This ensures you can view traffic, metrics and graphs for resources deployed in other namespaces.
 
 To limit scraping to specific namespaces, set `prometheus.prometheusSpec.ignoreNamespaceSelectors=true` and add one of the following configurations to ensure you can continue to view traffic, metrics and graphs for your deployed resources. 
 
 1. Add a Service Monitor or Pod Monitor in the namespace with the targets you want to scrape.
 1. Add an additionalScrapeConfig to your rancher-monitoring instance to scrape all targets in all namespaces.
 
+####  External Services
```
Comment (cbron) [inline packages/rancher-istio/charts/README.md:31]: I think we should specify that these are all for kiali.

File: packages/rancher-istio/charts/app-readme.md [hunk 4]
```diff
@@ -23,9 +23,23 @@ To limit scraping to specific namespaces, set `prometheus.prometheusSpec.ignoreN
...
 ```
 http://{{ .Values.nameOverride }}-prometheus.{{ .Values.namespaceOverride }}.svc:{{ prometheus.service.port }}
 ```
+**Custom Grafana Installation with Kiali**
+
+To use a custom Grafana installation, set the `kiali.external_services.grafana` url in the values.yaml. This url depends on the values for `nameOverride`, `namespaceOverride`, and `granfa.service.port` in your rancher-monitoring or other grafana instance:
+```
+http://{{ .Values.nameOverride }}-grafana.{{ .Values.namespaceOverride }}.svc:{{ grafana.service.port }}
+```
+**Custom Tracing Installation with Tracing**
```
Comment (cbron) [inline packages/rancher-istio/charts/app-readme.md:36]: with Kiali ?

Comment (brendarearden) [reply packages/rancher-istio/charts/app-readme.md:36]: Its an external link to the grafana instance from kiali, similar to the prometheus one. I was seeing errors in the ui without this set.

File: packages/rancher-istio/charts/README.md (hunk 3 above)
Comment (brendarearden) [reply packages/rancher-istio/charts/README.md:31]: If you look at it generated, the largest header is Kiali, and everything under it is smaller heading to indicate that its related to Kiali. Should we still make each sub heading more specific?

File: packages/rancher-istio/charts/app-readme.md (hunk 4 above)
Comment (cbron) [reply packages/rancher-istio/charts/app-readme.md:36]: I mean all of these say "custom blah install with kiali", not "with tracing".

File: packages/rancher-istio/charts/README.md (hunk 3 above)
Comment (cbron) [reply packages/rancher-istio/charts/README.md:31]: Ok, thats fine

**Review by cbron — APPROVED** [review]

**Pull request merged**